runtimebp/cpu.go

```diff
@@ -22,6 +22,8 @@ import (
 // If the current process is not running inside a container,
 // or if there's no limit set in cgroup,
 // it will fallback to runtime.NumCPU() instead.
+//
+// Deprecated: NumCPU is deprecated. See runtimebp/internal/maxprocs.
```

Comment:
Routing to an internal package is a bit confusing -- callers can't _use_ the internal package, so do you mean "see the docs in"?  If so, maybe the relevant docs should be in this package instead.

Either way, you can actually link to pkg.go.dev if you want since this is open source